go.mod

```diff
@@ -42,4 +36,63 @@ require (
 	humungus.tedunangst.com/r/gerc v0.1.2
 )
 
-go 1.13
+require (
+	github.com/DataDog/datadog-agent/pkg/obfuscate v0.0.0-20211129110424-6491aa3bf583 // indirect
+	github.com/DataDog/datadog-go v4.8.2+incompatible // indirect
```

Comment:
do you need to run go mod tidy? I would suspect we do directly depend on this.

Reply:
I ran it again and it correctly picked this up as required. 
I also bumped this to go 1.17 to get some of the [module improvements](https://go.dev/ref/mod#:~:text=This%20extra%20information%20enables%20module%20graph%20pruning%20and%20lazy%20module%20loading)